toolboxes/fluid/cases/rigid_body_paper_tests/naca0012_profile/naca0012_profile.cfg

```diff
@@ -0,0 +1,67 @@
+
+directory=toolboxes/fluid/new_testcases/naca0012_profile
```

Comment:
why add new_testcases?  should be : toolboxes/fluid/moving_body/naca0012_profile

Reply:
fixed